packages/my-extended-retail-app/package.json

```diff
@@ -87,5 +87,6 @@
     "iOS >= 9.0",
     "Android >= 4.4.4",
     "last 4 ChromeAndroid versions"
-  ]
+  ],
+  "private": "true"
```

Comment:
FYI this extended-retail-app package would be deleted in another PR #1205. So in this PR, I simply make the package private.

Reply:
I've deleted `my-extend-retail-app` in `feature/template-extensibility` I think we're ready to do that here as well right?